Mathlib/AlgebraicGeometry/PrimeSpectrum/Basic.lean

```diff
@@ -844,6 +849,34 @@ lemma exists_idempotent_basicOpen_eq_of_isClopen {s : Set (PrimeSpectrum R)}
 @[deprecated (since := "2024-11-11")]
 alias exists_idempotent_basicOpen_eq_of_is_clopen := exists_idempotent_basicOpen_eq_of_isClopen
 
+section localization_quotient
+variable [CommSemiring R] [CommSemiring S]
+
+open Localization Polynomial in
+lemma comap_C_eq_comap_localization_union_comap_quotient
+    {R : Type*} [CommRing R] (s : Set (PrimeSpectrum R[X])) (c : R) :
+    comap C '' s =
+      comap (algebraMap R (Away c)) '' (comap C ''
+        (comap (mapRingHom (algebraMap R (Away c))) ⁻¹' s)) ∪
+      comap (Ideal.Quotient.mk (.span {c})) '' (comap C ''
+        (comap (mapRingHom (Ideal.Quotient.mk _)) ⁻¹' s)) := by
```

Comment:
The title should say "localization away from `c`" instead of "at `c`". We localized at a prime ideal, but away from an element.